file/lib/file_source_endpoint.ex

```diff
@@ -142,36 +142,14 @@ defmodule Membrane.RTC.Engine.Endpoint.File do
     actions =
```

Comment:
please add infromation in module docs that this module now is only compatible with engine that has only one track